app/templates/file.hbs

```diff
@@ -2,14 +2,13 @@
 	{{!TODO handle video player XW-2605}}
 	{{!TODO files without thumbnails}}
 	{{#article-wrapper
-		articleRendered='articleRendered'
 		displayEmptyArticleInfo=false
 		model=model
-		openLightbox='openLightbox'
 		showComments=false
 		siteName=siteName
-		toggleDrawer='toggleDrawer'
 		wikiPageHeaderSubtitle=(i18n 'file-page.subtitle')
+		openLightbox=(action 'openLightbox' target=application)
```

Comment:
do we keep alphabetical order in component properties? Lint it! ;)

Reply:
not yet, but I think we will :)